platforms/quorum/configuration/roles/create/certificates/nodes/tasks/main.yaml

```diff
@@ -0,0 +1,10 @@
+# This role calls for ambassador certifacate creation for each node.
+---
+- name: Create Ambassador certificates
+  include_role: 
+    name: "create/certificates/ambassador"
```

Comment:
why is this role needed? Does not seem to do anything other than calling create/certificates/ambassador.